packages/identifier/src/reserved.ts

```diff
@@ -1047,30 +1047,29 @@ const famousAccounts = [
   'zerohora',
 ]
 
-// Needs additional work to add more words that should be caught and regex added to catch common variations
+// naive, incomplete list of slurs and unacceptable words
 const slurs = [
-  'chinaman',
-  'chinamen',
```

Comment:
...hate to spam up this already heavily traffic'd commit, but why were these removed?

Reply:
Also / on the same note 「 jap 」 and 「 japs 」

Reply:
reverted in #1332

Reply:
No, #1333 was closed without being merged. I don't think we'll ever know the reason why this removal was made.